src/routes/+page.svelte

```diff
@@ -33,7 +35,7 @@
 			...history,
 			{
 				type: ChatHistoryType.Human,
-				content: userInput
+				content: sanitizeHtml(userInput)
```

Comment:
I don't know if we want to do this here, we may want to do it on the backend and just incorporate it inside of the ChatCompletion class

Reply:
I can make it only backend but the clensed message would not display on the frontend. The chatbot would get the clensed text but the display will not.